tools/indexers/trials.js

```diff
@@ -332,11 +332,16 @@ const trialMapping = {
     },
     discrepancies: {
       properties: {
-        public_title: getDiscrepancyRecordMapping({
```

Comment:
@vitorbaptista I updated these attributes to bring them in line with [`discrepancyFields` in `trial` model](https://github.com/opentrials/api/blob/master/api/models/trial.js#L187). I already run `reindex` with this mapping on a production dump and it didn't raise errors which means these fields (`public_title` etc.) are not used anymore. Can you remember any reason for them?